pkg/kritis/attestation/pgpkey.go

```diff
@@ -96,7 +90,11 @@ func parsePrivateKey(privateKey string) (*packet.PrivateKey, error) {
 }
 
 func parseKey(key string, keytype string) (packet.Packet, error) {
-	r := strings.NewReader(key)
+	pemKeyString, err := base64.StdEncoding.DecodeString(key)
```

Comment:
When naming variables, omit words that are obvious given a variable's type declaration.

In Go, it's idiomatic to prefer short names for such a short-lived variable, like "s".